### DISCUSSION TITLE
shell and sphere model not finding particles on 2nd cell

### DISCUSSION THREAD

**Discussion by shimwell** [svalinn/DAGMC#889] [Q&A] [answered]
I've been making a model of nested spheres as a test case

![Screenshot from 2023-07-25 20-57-13](https://github.com/svalinn/DAGMC/assets/8583900/96abb9a2-9e33-4760-8667-b3f0f473a376)

It has just two surfaces. When transporting particles in openmc through the sphere I can't get a flux tally on the outer sphere.

It looks like particles are traveling through the geometry and not interacting.

I made the model with cad-to-dagmc so clearly I've got something wrong with the surfaces.

h5m model and openmc xml files attached [sphericalshell.zip](https://github.com/svalinn/DAGMC/files/12165203/sphericalshell.zip)

Model is part of this github repos of models
https://github.com/fusion-energy/model_benchmark_zoo

Script to reproduce the model is here, but [installation](https://github.com/fusion-energy/model_benchmark_zoo/blob/main/README.md) of packages is needed to fully reproduce
https://github.com/fusion-energy/model_benchmark_zoo/blob/main/examples/compare_csg_cad_sphericalshell.py

I would be interested to know if I've missed and surface sense on the dagmc model or anything like that?

**Comment by makeclean** [comment, marked as answer]
Your inner surface has the wrong sense

![image](https://github.com/svalinn/DAGMC/assets/2439754/08093d3e-fb2f-495b-9e64-0a897081df8f)

**Reply by shimwell** [reply]
Many thanks @makeclean I would be fascinated to know how you made that plot. 👀
